bin/create-gxa-analytics-schema.sh

```diff
@@ -27,6 +27,101 @@ curl -X POST -H 'Content-type:application/json' --data-binary '{
 
+printf "\n\nDelete field is_private\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "delete-field":
+  {
+    "name": "is_private"
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+printf "\n\nCreate field is_private (boolean)\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "add-field":
+  {
+    "name": "is_private",
+    "type": "boolean"
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+#############################################################################################
+
+printf "\n\nDelete copy field rule bioentity_identifier -> bioentity_identifier_search\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "delete-copy-field":
+  {
+    "source": "bioentity_identifier",
+    "dest": "bioentity_identifier_search"
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+printf "\n\nDelete field bioentity_identifier\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "delete-field":
+  {
+    "name": "bioentity_identifier"
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+printf "\n\nCreate field bioentity_identifier (string, DocValues)\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "add-field":
+  {
+    "name": "bioentity_identifier",
+    "type": "string",
+    "docValues": true
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+printf "\n\nDelete field bioentity_identifier_search\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "delete-field":
+  {
+    "name": "bioentity_identifier_search"
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+printf "\n\nCreate field bioentity_identifier_search (lowercase)\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "add-field":
+  {
+    "name": "bioentity_identifier_search",
+    "type": "lowercase"
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+printf "\n\nAdd copy field rule bioentity_identifier -> bioentity_identifier_search\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "add-copy-field":
+  {
+    "source": "bioentity_identifier",
+    "dest": "bioentity_identifier_search"
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+#############################################################################################
+
+printf "\n\nDelete field assay_group_id\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "delete-field" :
+  {
+    "name": "assay_group_id"
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+printf "\n\nCreate field assay_group_id (string, DocValues)\n"
+curl -X POST -H 'Content-type:application/json' --data-binary '{
+  "add-field":
+  {
+    "name": "assay_group_id",
+    "type": "string",
+    "docValues": true
+  }
+}' http://${HOST}/solr/${CORE}/schema
+
+#############################################################################################
+
+# TODO Do we need assay? is not in https://github.com/ebi-gene-expression-group/solr-bulk/blob/master/bin/create-bulk-analytics-schema.sh
```

Comment:
@alfonsomunozpomer - can you please confirm this? I'm not sure about this.

Reply:
We can as well leave these as a transition, and start filling both.